index.bs

```diff
@@ -885,7 +885,13 @@ Composition of the Advisory Board</h5>
 	the [=Team=] appoints the Chair,
 	who <em class=rfc2119>should</em> choose a co-chair among the elected participants.
 	The Chair(s) are subject to ratification by secret ballot
-	by two thirds of the AB upon appointment and at the start of each AB term.
+	by two thirds of the AB upon appointment.
+	Chair selection <em class=rfc2119>must</em> be run
+	at least at the start of each regular term,
+	as well as when a majority of the participants request it;
+	and <em class=rfc2119>may</em> be run at other times when initiated by the current chairs or the Team,
+	for example if a chair steps down or if a minority of the participants make such a request.
```

Comment:
How small a minority? only one? who decides if it's a large enough minority? I would delete this last phrase "("or if") as it raises more questions than it answers (and below).

Reply:
That's up to the Team (or the current chairs).  They're not required to do so.